.gitlab-ci.yml

```diff
@@ -2,7 +2,7 @@ variables:
   TYPE: app
   MULTI_ARCH: "true"
 
-image: golang:1.20
+image: golang:1.21.11
```

Comment:
don't you think 1:21 would be okay? i dont see the benefit of specifying the patch version here